Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -112,11 +142,13 @@ gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateFontSize = function() {
   this._pixiObject.dirty = true;
 };
 
+// FIXME position et placement correspondent pas à l'IDE
```

Comment:
Maybe something related to the anchor which should be set to 0.5, 0.5 (so that the object rotates from its center)?